README.md

```diff
@@ -54,15 +54,15 @@ View guide at: https://luckyframework.org/guides/browser-tests/
 
 ## Contributing
 
-1.  Fork it ( https://github.com/luckyframework/lucky_flow/fork )
-1.  Create your feature branch (git checkout -b my-new-feature)
-1.  Install docker and docker-compose: https://docs.docker.com/compose/install/
-1.  Run `bin/setup`
-1.  Make your changes
-1.  Run `bin/test` to run the specs, build shards, and check formatting
-1.  Commit your changes (git commit -am 'Add some feature')
-1.  Push to the branch (git push origin my-new-feature)
-1.  Create a new Pull Request
+1. Fork it ( https://github.com/luckyframework/lucky_flow/fork )
+2. Create your feature branch (git checkout -b my-new-feature)
+3. Install docker and docker-compose: https://docs.docker.com/compose/install/
+4. Run `script/setup`
+5. Make your changes
+6. Run `script/test` to run the specs, build shards, and check formatting
+7. Commit your changes (git commit -am 'Add some feature')
+8. Push to the branch (git push origin my-new-feature)
+9. Create a new Pull Request
```

Comment:
Could you change the numbers back? It looks a bit odd, but the markdown renderer will order them correctly. The advantage of using all `1` is we can add steps in-between, re-order, etc. and it'll always have the right order when output in the README

Reply:
Not sure what editor @matthewmcgarvey is using, but I know some editors will automatically order this stuff without you asking. It's annoying IMO but it may have just been a mistake. I do like when the actual markdown text is formatted nicely because that's a large part of why markdown is nice: it looks good both in code and formatted. But agreed that in this case it's a pain to try and keep these all up to date all the time.

Reply:
I'm using a markdown linter. It's screaming about several places in the page but just changed that one since it surprised me. Didn't realize till after that it is still rendered correctly.